SevenZip.sln

```diff
@@ -1,40 +1,40 @@
-﻿
```

Comment:
Not really a problem, but a little odd to include what I guess is whitespace changes in a PR 🤷

Reply:
ok. I will fix it